angular/TreeView/Getting-Started.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Getting Started for Angular TreeView
-description: Getting Started for Angular TreeView
+title: Getting Started with Angular TreeView Control | Syncfusion
+description: Learn here about getting started with Angular TreeView in Syncfusion Essential Angular Treeview Control, its elements, and more.
```

Comment:
You have duplicated controlname and platformname. Please check in all places and use as above screenshot.
![image](https://github.com/syncfusion-content/angular-docs/assets/95272306/49ba654c-71bb-48ed-8395-f24b2939fd44)

Reply:
Done